cli/archive/Tutorial/tutorial.html

```diff
@@ -500,8 +500,8 @@ <h3 id="unnesting-nested-collections-using-join">Unnesting Nested Collections Us
 <p>One may think that the <code>FROM</code> clause of the example executes, in a sense, a <code>JOIN</code> between employees and projects. If it helps you to think in terms of <code>JOIN</code>, you may replace the comma with <code>JOIN</code>. That is, the following two queries are equivalent.</p>
```

Comment:
`tutorial.html` and `tutorial.pdf` were generated through the docker process outlined [here](https://github.com/partiql/partiql-lang-kotlin/tree/ce06bcf8203ea963596fc723e0bb7c5d1b422dfd/docs/Docker). These two files can mostly be ignored as their changes are documented in `tutorial.md`.